newsroom/wire/search.py

```diff
@@ -289,7 +289,11 @@ def get_product_items(self, product_id, size):
         if not product:
             return
 
-        query['bool']['should'] = []
+        query['bool']['should'] = [
```

Comment:
I guess this should be part of must query otherwise if there is product query or product id both might be false and it will pass